scripts/models/patrol/patrol_schema_item.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+from typing import List, Union, Optional, Dict, Annotated
+
+from pydantic import BaseModel, Field, StringConstraints
+
+from scripts.models.common.relationship_status import RelationshipStatus
+from scripts.models.common.skill import Skill
+from scripts.models.patrol.biome import BiomeEnum
+from scripts.models.patrol.outcome import Outcome
+from scripts.models.patrol.patrol_tag import PatrolTag
+from scripts.models.patrol.patrol_type import PatrolType
+from scripts.models.common.season import SeasonEnum
+
+
+class PatrolSchemaItem(BaseModel):
+    patrol_id: str = Field(
+        ..., description="Unique string used to identify the patrol."
+    )
+    biome: List[BiomeEnum] = Field(
+        ..., description="Controls the biome(s) the patrol appears in"
+    )
+    season: List[SeasonEnum] = Field(
+        ..., description="Controls the season(s) the patrol appears in."
+    )
+    types: List[PatrolType] = Field(..., description="Controls the type of patrol.")
+    tags: List[
+        Union[PatrolTag, Annotated[str, StringConstraints(pattern=r"^clan:(.+)$")]]
+    ] = Field(
+        ...,
+        description="Tags are used for some filtering purposes, and some odd-and-ends. Tags never affect outcome.",
+    )
+    patrol_art: Optional[str] = Field(
+        ...,
+        description="The name of displayed patrol art file, without any file extension (no .png).",
+    )
+    patrol_art_clean: Optional[str] = Field(
+        None,
+        description='If patrol_art contains gore, this line can hold a clean version. The existence of a non-empty string in this parameter marks the patrol art in "patrol_art" as explicit.',
+    )
+    min_cats: int = Field(
+        ..., description="Minimum total number of cats for this patrol."
+    )
+    max_cats: int = Field(
+        ..., description="Maximum total number of cats for this patrol"
+    )
+    min_max_status: Optional[Dict[str, List[int]]] = Field(
```

Comment:
```suggestion
    min_max_status: Optional[Dict[str, Tuple[int, int]]] = Field(
```
Also, the keys should be one of: 
```
            "medicine cat",
            "warrior",
            "leader",
            "deputy",
            "apprentice",
            "medicine cat apprentice",
            "healer cats",
            "normal adult",
            "all apprentices"
```